src/dsverifier.cpp

```diff
@@ -2615,20 +3594,41 @@ void extract_data_from_ss_file()
       }
       else if(current_line[i] == ';')
       {
-        _controller.K[lines][columns] = std::stod(str_bits);
+        if(isNumber(str_bits))
+        {
+          _controller.K[lines][columns] = std::stod(str_bits);
+        }
+        else
+        {
+        // TODO: to be implemented
```

Comment:
What should be implemented here? Can you please be more specific?

Reply:
DONE!